apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md

```diff
@@ -516,18 +516,22 @@ data-synthetic/Sample-A-Synthetic-Data/ based on data from Step 1.  Changed the
 12.  Uploaded all of these files in Test, Prod-sandbox, and Prod.
 
 ### Synthea Data Set Details
-09/09/2021 - Generated Synthea data set from the following properities file that is located at: `[./synthea/synthea.properities-08092021](./synthea/synthea.properities-08092021)`.
-
-exporter.bfd.export = true
-exporter.bfd.config_file = export/bfd_field_values.tsv
-exporter.bfd.bene_id_start = -10000000000000
-exporter.bfd.clm_id_start = -10000000000000
-exporter.bfd.clm_grp_id_start = -100000000
-exporter.bfd.pde_id_start = -10000000000
-exporter.bfd.mbi_start = 1S00-E00-AA01 - the next mbi start should be 1S00-E00-FZ01
-exporter.bfd.hicn_start = T01000000A
-exporter.bfd.partd_contract_start = Z0001
+01/18/2022 - Generated Synthea data set from the following properities file that is located at: `[./synthea/synthea.properities-01182022](./synthea/synthea.properities-01182022)`.
+
+exporter.bfd.export = false
+exporter.bfd.require_code_maps = true
+exporter.bfd.bene_id_start = -10000000010000
+exporter.bfd.clm_id_start = -10000003175478
+exporter.bfd.clm_grp_id_start = -100010000
```

Comment:
I'm curious how this works. do we only have one claim per group? and how does synthea know how many claims groups to generate? I didn't see a setting for the number of groups.